diann/1.9.1/Dockerfile

```diff
@@ -0,0 +1,45 @@
+################## BASE IMAGE ######################
+FROM ubuntu:latest
+
+################## METADATA ######################
+LABEL base_image="ubuntu:latest"
+LABEL version="1"
+LABEL software="diann"
+LABEL software.version="1.9.1"
+LABEL about.summary="DIA-NN - a universal software for data-independent acquisition (DIA) proteomics data processing."
+LABEL about.home="https://github.com/vdemichev/DiaNN"
+LABEL about.documentation="https://github.com/vdemichev/DiaNN"
+LABEL about.license_file="https://github.com/vdemichev/DiaNN/LICENSE.txt"
+LABEL about.license="Proprietary - See LICENSE.txt for details"
+LABEL about.tags="Proteomics"
+
+################## MAINTAINER ######################
+MAINTAINER Niko Pinter <niko.pinter@uniklinik-freiburg.de>
+
+################## INSTALLATION ######################
+
+ENV DEBIAN_FRONTEND noninteractive
+
+## Update and install packages
+RUN apt-get update -y && apt-get install -y \
+    g++ build-essential cmake zlib1g-dev libbz2-dev libboost-all-dev wget locales unzip \
+ && sed -i '/en_US.UTF-8/s/^# //g' /etc/locale.gen \
+ && locale-gen
```

Comment:
Since you are using a pre-compiled binary, do you need all these libs?
(This is minor, but might as well save some space..)

Reply:
I took this line from the recipe of v1.8.1. I'll try to break it down to a minimum. Thank you!